auth/auth.go

```diff
@@ -145,3 +177,29 @@ func (a *AuthService) Refresh(w http.ResponseWriter, r *http.Request) {
 		invalidRefreshToken(w)
 	}
 }
+
+func (a *AuthService) oAuthSignUp(data oAuthData, w http.ResponseWriter) {
+	u, findErr := a.client.User.Query().Where(user.EmailEQ(data.GetEmail())).Only(a.ctx)
```

Comment:
`data.GetEmail()` is called twice, I would put it into a variable

Reply:
fixed